src/validation.h

```diff
@@ -93,7 +93,7 @@ static const unsigned int DEFAULT_DESCENDANT_LIMIT = 25;
 /** Default for -limitdescendantsize, maximum kilobytes of in-mempool descendants */
 static const unsigned int DEFAULT_DESCENDANT_SIZE_LIMIT = 101;
 /** Default for -mempoolexpiry, expiration time for mempool transactions in hours */
-static const unsigned int DEFAULT_MEMPOOL_EXPIRY = 1;
+static const unsigned int DEFAULT_MEMPOOL_EXPIRY = 24;
```

Comment:
I propose to increase it here for another couple of hours in order to prevent a situation when the transaction time has come to be included in the block, but it will be deleted by timeout

Reply:
I agree. [Set it in 26](https://github.com/pocketnetteam/pocketnet.core/commit/2076d11c951f2ca567999c34eff4382f43bcceab)